Makefile

```diff
@@ -44,3 +47,6 @@ distclean:: delete-test/system_SUITE_data/samples.zip
 
 include rabbitmq-components.mk
```

Comment:
This will override the versions you set above (last one wins).

Reply:
You can move the include before your dep_ lines or move the RabbitMQ dep lines between this include and the include erlang.mk to fix.